examples/fractal.cpp

```diff
@@ -22,7 +22,7 @@
 
 #include <fstream>
 
-#include <math.h>
+#include <cmath>
```

Comment:
In GCC, if we keep math.h, then the inline `lerp` defined below is confused with std::lerp from C++20